BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -271,23 +271,6 @@ class BraintreeClient internal constructor(
         )
     }
 
-    /**
-     * Set this property to true to allow the SDK to handle deep links on behalf of the host
-     * application for browser switched flows.
-     *
-     * For web payment flows, this means launching the browser in a task separate from the calling
-     * activity.
-     *
-     * NOTE: When this property is set to true, all custom url schemes set in [BraintreeClient]
-     * constructors will be ignored.
-     *
-     * @param launchesBrowserSwitchAsNewTask set to true to allow the SDK to capture deep links.
-     * This value is false by default.
-     */
-    fun launchesBrowserSwitchAsNewTask(launchesBrowserSwitchAsNewTask: Boolean) {
```

Comment:
In v5 BraintreeClient was made internal. So merchants lost the ability to set this flag.